docs/live-query.rst

```diff
@@ -125,6 +125,34 @@ The sequence of calls are:
     >>> finished_job.get_output_as_file("/Users/myname/mydir/livequeryresults_async.csv")
 
 
+Scroll results
+--------------
+
+If you would like to ingest all the Live Query results whether that be from one Run or multiple Runs consider using the scroll option
+to fetch the latest results. The scroll option is limited to the last 24 hours for results across all Runs. You either need to specify
+a time_received or a list of one or more Run ids
```

Comment:
This line should end with a :: (double colon) so the indented paragraph below will be displayed in monospace.